std_execution.bs

```diff
@@ -2950,79 +2944,90 @@ namespace std::execution {
       <i>unspecified</i> just(Ts &&...) noexcept(<i>see below</i>);
     template&lt;<i>movable-value</i> Error>
       <i>unspecified</i> just_error(Error &&) noexcept(<i>see below</i>);
     <i>unspecified</i> just_done() noexcept;
     template&lt;<i>movable-value</i>... Ts>
       <i>unspecified</i> just(Ts &&...);
-    struct transfer_just_t;
-    inline constexpr transfer_just_t transfer_just{};
+    inline constexpr <i>unspecified</i>::transfer_just_t transfer_just{};
 
     // [execution.senders.adaptors], sender adaptors
     template&lt;<i>class-type</i> D>
       struct sender_adaptor_closure { };
 
-    struct on_t;
-    inline constexpr on_t on{};
-    struct transfer_t;
-    inline constexpr transfer_t transfer{};
-    struct schedule_from_t;
-    inline constexpr schedule_from_t schedule_from{};
+    inline constexpr <i>unspecified</i>::on_t on{};
+    inline constexpr <i>unspecified</i>::transfer_t transfer{};
+    inline constexpr <i>unspecified</i>::schedule_from_t schedule_from{};
 
-    struct then_t;
-    inline constexpr then_t then{};
-    struct upon_error_t;
-    inline constexpr upon_error_t upon_error{};
-    struct upon_done_t;
-    inline constexpr upon_done_t upon_done{};
+    inline constexpr <i>unspecified</i>::then_t then{};
+    inline constexpr <i>unspecified</i>::upon_error_t upon_error{};
+    inline constexpr <i>unspecified</i>::upon_done_t upon_done{};
 
-    struct let_value_t;
-    inline constexpr let_value_t let_value{};
-    struct let_error_t;
-    inline constexpr let_error_t let_error{};
-    struct let_done_t;
-    inline constexpr let_done_t let_done{};
+    inline constexpr <i>unspecified</i>::let_value_t let_value{};
+    inline constexpr <i>unspecified</i>::let_error_t let_error{};
+    inline constexpr <i>unspecified</i>::let_done_t let_done{};
 
-    struct bulk_t;
-    inline constexpr bulk_t bulk{};
+    inline constexpr <i>unspecified</i>::bulk_t bulk{};
 
-    struct split_t;
-    inline constexpr split_t split{};
-    struct when_all_t;
-    inline constexpr when_all_t when_all{};
-    struct when_all_with_variant_t;
-    inline constexpr when_all_with_variant_t when_all_with_variant{};
-    struct transfer_when_all_t;
-    inline constexpr transfer_when_all_t transfer_when_all{};
-    struct transfer_when_all_with_variant_t;
-    inline constexpr transfer_when_all_with_variant_t
+    inline constexpr <i>unspecified</i>::split_t split{};
+    inline constexpr <i>unspecified</i>::when_all_t when_all{};
+    inline constexpr <i>unspecified</i>::when_all_with_variant_t when_all_with_variant{};
+    inline constexpr <i>unspecified</i>::transfer_when_all_t transfer_when_all{};
+    inline constexpr <i>unspecified</i>::transfer_when_all_with_variant_t
       transfer_when_all_with_variant{};
 
     template&lt;typed_sender S>
       using <i>into-variant-type</i> = <i>see-below</i>; // exposition-only
     template&lt;typed_sender S>
       <i>see-below</i> into_variant(S &&);
```

Comment:
`into_variant` should live in its own namespace, or with the `just*` functions whichever.